x-kiosk-imx8/Dockerfile

```diff
@@ -0,0 +1,225 @@
+                       python pkg-config p7zip openbox mesa-common-dev \
+                       libxtst6 libxtst-dev libxt-dev libxss-dev libxslt1-dev \
+                       libxshmfence-dev libxrender1 libxrandr2 libxkbcommon-dev \
+                       libxinerama1 libxi6 libxfixes3 libxdamage1 libxcursor1 \
+                       libxcomposite1 libx11-xcb1 libwww-perl \
+                       libwayland-egl1-mesa libvulkan1 libvulkan-dev libva-dev \
+                       libudev-dev libssl-dev libsqlite3-dev libspeechd2 \
+                       libspeechd-dev libsctp-dev libpulse0 libpulse-dev \
+                       libpng16-16 libpixman-1-0 libpci3 libpci-dev \
+                       libpango-1.0-0 libnss3-dev libnss3 libnspr4-dev \
+                       libnspr4 libkrb5-dev libjpeg-dev libinput10 libinput-dev \
+                       libgtk-3-dev libgtk-3-0 libglu1-mesa-dev libglib2.0-dev \
+                       libglib2.0-0 libgbm1 libgbm-dev libfreetype6 \
+                       libfontconfig1 libffi-dev libevdev2 libevdev-dev \
+                       libelf-dev libdrm2 libdrm-dev libcurl4-gnutls-dev \
+                       libcups2-dev libcups2 libcap2 libcap-dev libcairo2-dev \
+                       libcairo2 libc6-dev libbz2-dev libbrlapi-dev \
+                       libbluetooth-dev libatspi2.0-dev libatspi2.0-0 \
+                       libatk1.0-0 libasound2-dev libasound2 \
+                       libappindicator3-dev libappindicator3-1 gperf flex \
+                       fakeroot elfutils devscripts dbus-x11 cdbs bison \
+                       binutils-arm-linux-gnueabihf binutils-aarch64-linux-gnu \
+                       apache2-bin libflac-dev flac lsb-release wget \
+                       software-properties-common dirmngr apt-transport-https ca-certificates \
+    && apt-get clean && apt-get autoremove && rm -rf /var/lib/apt/lists/*
+
+COPY setup_12.x /
+RUN bash /setup_12.x
+RUN apt-get --purge remove -y gcc g++ cpp
+RUN add-apt-repository -y ppa:ubuntu-toolchain-r/test
+
+RUN apt-get update && \
+    apt-get install -y gcc-10 g++-10 nodejs && \
+    rm -rf /var/lib/apt/lists/*
+
+COPY llvm.sh /
+RUN bash /llvm.sh
+
+COPY create-links.sh /
+RUN bash /create-links.sh
+
+# Install Chromium
+
+RUN git clone https://chromium.googlesource.com/chromium/tools/depot_tools.git && cd depot_tools && git checkout 8dd74d4f85a739ff883831960c665b0a3a24ec36
+ENV PATH="/depot_tools:${PATH}"
+
+RUN mkdir /chromium
+WORKDIR /chromium
+RUN wget -q https://commondatastorage.googleapis.com/chromium-browser-official/chromium-96.0.4664.110.tar.xz 
+RUN tar xf chromium-96.0.4664.110.tar.xz
+
+WORKDIR /root
+
+ENV CXX=g++
+RUN git clone https://github.com/ninja-build/ninja.git -b v1.8.2 && cd ninja && git checkout 41156cd9c97a4e49473c473b7952f8cb7c43c56c
+RUN cd ninja && ./configure.py --bootstrap
+ENV PATH="/root/ninja:${PATH}"
+
+RUN \
+    rm /chromium/chromium-96.0.4664.110/third_party/depot_tools/ninja && \
```

Comment:
I'd suggest adding `-f` for any `rm` in a non-interactive context. Here and below.